src/tls_mbed.c

```diff
@@ -66,7 +66,7 @@ static int mbed_rng(void *ctx, unsigned char *buf, size_t len) {
 
 static void debug_cb(void *c, int lev, const char *s, int n, const char *s2) {
   n = (int) strlen(s2) - 1;
-  MG_VERBOSE(("%lu %d %.*s", ((struct mg_connection *) c)->id, lev, n, s2));
+  MG_INFO(("%lu %d %.*s", ((struct mg_connection *) c)->id, lev, n, s2));
```

Comment:
Changing _VERBOSE to _INFO inside a "debug" callback raised my eyebrows.
Intended or test leftover ?

Reply:
That's fine, cause if this callback is called, we want to see what it prints.